python/pysrc/kaskada/_timestream.py

```diff
@@ -770,6 +790,12 @@ def min(self, window: Optional[kd.windows.Window] = None) -> Timestream:
 
         Args:
             window: The window to use for the aggregation. Defaults to the entire Timestream.
+
+        See Also:
+            This returns the the minimum of values in a
+            column up to and including the current row. See the
+            comparison :func:`least` to get the least value
```

Comment:
comparison

Reply:
i ended up removing `the comparison`.  now it is just `see *least* to get minimum value...`